vidarrtest-regression.json.in

```diff
@@ -102,7 +102,7 @@
             {
                 "metrics_calculate": "@CHECKOUT@/tests/calculate.sh",
                 "metrics_compare": "@CHECKOUT@/tests/compare.sh",
-                "output_metrics": "/.mounts/labs/gsi/testdata/insertSizeMetrics/output_metrics/no_output_prefix.metrics",
```

Comment:
I would keep this unchanged. The title contains info which is useful for development, at the same time it is not something sensitive that needs to be hidden